online-inference/tensorizer-isvc/benchmark/load_test_sync.py

```diff
@@ -0,0 +1,60 @@
+import random
+import requests
+import time
+from argparse import ArgumentParser
+
+# Uncomment to use for Locust
+
+# from locust import HttpUser, task
+
+# class QuickstartUser(HttpUser):
+#     @task
+#     def predict(self):
+#         inputs = ["Hello, how are you?", "What up dig dog?", "You are a killer!", "JCPenny is a bad store", "Life is great"]
+
+
+#         with self.client.get(f"/predict/{random.choice(inputs)}") as response:
+#             if response.status_code != 200:
+#                 response.failure("Could not return response")
+
+inputs = ["Hello, how are you?", "What up dig dog?", "You are a killer!", "JCPenny is a bad store", "Life is great", "Chilling on a roof", "Love you", "Mox is cute", "You are my enemy", "Change is required", "Love the life"]
```

Comment:
JCPenney has a typo in it, but maybe we should avoid picking fights with still-existing companies just for an example string, haha.